main.py

```diff
@@ -1,5 +1,35 @@
 import init_django_orm  # noqa: F401
+from db.models import Genre, Actor
 
 
+# Shitcode alert!
 def main():
-    pass
+    # Create
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
+    Actor.objects.create(first_name="George", last_name="Klooney")
```

Comment:
You can do it using loops. Imagine tomorrow your boss we bring more data to save into DB)